flake.nix

```diff
@@ -58,13 +58,18 @@
           };
 
         build = forAllSystems (system: nixpkgsFor.${system}.patchelf-new);
+        build-sanitized = forAllSystems (system: nixpkgsFor.${system}.patchelf-new.overrideAttrs (old: {
```

Comment:
@Luis-Hebendanz any other sanitizer we should add?

Reply:
Ah yes I would add "-fsanitize=integer" too to catch unsigned integer overflow and integer implicit unsigned integer truncation :)

Reply:
Oh. Is this not covered yet by the undefined behavior sanitizer?

Reply:
No because unsigned integer overflow & truncation is not undefined behavior :)